flight_statistics.py

```
@@ -0,0 +1,528 @@
+    draw.text((7, 4), "Flight records", fill="black", font=header)
+    draw.text(
+        (7, 21), f"{records['encounters']} recorded flybys", fill="black", font=detail
+    )
+    draw.line([(7, 34), (image.width - 7, 34)], fill="black", width=1)
+    rows = []
+    for title in ("oldest", "youngest"):
+        item = records[title]
+        if item:
+            rows.append((title.title(), item["label"], str(item["year"])))
+    if records["repeat"]:
+        rows.append(("Most seen", records["repeat"][0], f"×{records['repeat'][1]}"))
```

Ruff (0.15.21)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)